### PR TITLE
Add collection playground project

### PR DESCRIPTION
**Pull request by yigit**
Note that I decided to not include collections2 since it is just an
infra project and we've decided not to merge it into collection as is
(instead do it piecemeal).

Bug: n/a
Test: gw bOS

### CODE REVIEW

**Comment by dlam** [comment]
Can you please update the README.md, CONTRIBUTING.md to include this project, as well as include buganizer link in the ISSUES template?

**Comment by yigit** [comment]
updated.

**Review by tikurahul — APPROVED** [review]

File: .github/workflows/presubmit.yml [hunk 1]
```diff
@@ -115,12 +115,12 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         # if you would like to remove some projects temporarily, use .github/ci-control/ci-config.json instead.
-        project: ["activity", "biometric", "compose-runtime", "datastore", "fragment", "lifecycle", "navigation", "paging", "room", "work"]
+        project: ["activity", "biometric", "collection", "compose-runtime", "datastore", "fragment", "lifecycle", "navigation", "paging", "room", "work"]
         include:
           - project: "compose-runtime"
             project-root: "compose/runtime"
-          - project: "navigation"
-            custom-os: "macos-latest" # run one of them on a mac to ensure mac setup works
```
Comment (liutikas) [inline .github/workflows/presubmit.yml:123]: I'd leave this comment since it is different from all the other runners

**Comment by 525411** [comment]
to yigit do u think that of all this is playground we try my best maybee we
not are educated like you..to treat us and listening evry one acting to
playgroud the status..we do respect to u..we are poor and literate are
happy

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -6,6 +6,9 @@ contact_links:
   - name: Biometric
     url: https://issuetracker.google.com/issues/new?component=559537&template=1214425
     about: File a bug or feature request for Biometric
+  - name: Collection
+    url: https://b.corp.google.com/issues/new?component=460756&template=1422573
```
Comment (msfjarvis) [inline .github/ISSUE_TEMPLATE/config.yml:10]:
```suggestion
    url: https://issuetracker.google.com/issues/new?component=460756&template=1422573
```

**Pull request closed without merging**